src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/SyaContactDetails.java

```diff
@@ -1,92 +1,16 @@
 package uk.gov.hmcts.reform.sscs.domain.wrapper;
 
-public class SyaContactDetails {
+import lombok.Data;
 
+@Data
+public class SyaContactDetails {
     private String addressLine1;
-
     private String addressLine2;
-
     private String townCity;
-
     private String county;
-
     private String postCode;
-
     private String phoneNumber;
-
     private String emailAddress;
-
-
-    public SyaContactDetails() {
-        // For JSON
-    }
-
-    public String getAddressLine1() {
-        return addressLine1;
-    }
-
-    public void setAddressLine1(String addressLine1) {
-        this.addressLine1 = addressLine1;
-    }
-
-    public String getAddressLine2() {
-        return addressLine2;
-    }
-
-    public void setAddressLine2(String addressLine2) {
-        this.addressLine2 = addressLine2;
-    }
-
-    public String getTownCity() {
-        return townCity;
-    }
-
-    public void setTownCity(String townCity) {
-        this.townCity = townCity;
-    }
-
-    public String getCounty() {
-        return county;
-    }
-
-    public void setCounty(String county) {
-        this.county = county;
-    }
-
-    public String getPostCode() {
-        return postCode;
-    }
-
-    public void setPostCode(String postCode) {
-        this.postCode = postCode;
-    }
-
-    public String getPhoneNumber() {
-        return phoneNumber;
-    }
-
-    public void setPhoneNumber(String phoneNumber) {
-        this.phoneNumber = phoneNumber;
-    }
-
-    public String getEmailAddress() {
-        return emailAddress;
-    }
-
-    public void setEmailAddress(String emailAddress) {
-        this.emailAddress = emailAddress;
-    }
-
-    @Override
-    public String toString() {
-        return "SyaContactDetails{"
-                + " addressLine1='" + addressLine1 + '\''
-                + ", addressLine2='" + addressLine2 + '\''
-                + ", townCity='" + townCity + '\''
-                + ", county='" + county + '\''
-                + ", postCode='" + postCode + '\''
-                + ", phoneNumber='" + phoneNumber + '\''
-                + ", emailAddress='" + emailAddress + '\''
-                + '}';
-    }
+    private String postcodeLookup;
```

Comment:
There is a difference with `postCode`, `postcodeLookup` and `postcodeAddress`.

I don't mind if `postCode` was changed to `postcode`.

Reply:
postCodeLookup and postcodeAddress are coming from the PostCodeLookUp service which is a third service to look up for the full address given a postcode. The postCode is fulfilled by that third service response. To sum up, there are not the same.

Reply:
and there must be side effects in CCD since the postCode is defined in CCD using camel naming convention too.